src/index.js

```diff
@@ -3,8 +3,14 @@
  * @author Brandon Mills
  */
 
+//-----------------------------------------------------------------------------
+// Imports
+//-----------------------------------------------------------------------------
+
 import { processor } from "./processor.js";
 
+
```

Comment:
```suggestion
//-----------------------------------------------------------------------------
// Exports
//-----------------------------------------------------------------------------
```

To fix the "More than 2 blank lines not allowed" lint error, and I'm guessing this was supposed to be the start of exports.